src/assets/icons/search.svg

```diff
@@ -1,6 +1,6 @@
 <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" width="24" height="24">
-  <g fill="none" stroke="currentColor" stroke-linecap="round">
-    <circle cx="9.5" cy="9.5" r="7.5" stroke-width="3"/>
-    <line x1="16" y1="16" x2="21" y2="21" stroke-width="4"/>
+  <g fill="none" stroke-linecap="round" stroke-miterlimit="10">
```

Comment:
Does `stroke-miterlimit` actually have a visible effect?

Reply:
Not that I could tell! Removed.